deploy/chart/devfile-registry/templates/_template.tpl

```diff
@@ -22,6 +22,15 @@
 {{- .Values.global.ingress.domain | printf "%s.%s" $hostname -}}
 {{- end -}}
 
+{{- define "devfileregistry.routeHostname" -}}
+{{- $hostname := .Values.hostnameOverride | default (printf "devfile-registry-%s" .Release.Namespace) -}}
```

Comment:
The route's hostname will be prefixed with `{{ template "devfileregistry.fullname" . }}`, which should default to `devfile-registry` for most cases. But to cover all grounds, should we change the default here to `{{ template "devfileregistry.fullname" . }}`?

Edit: I didn't see that the script below hardcoded the chart name to `devfile-registry`.

Reply:
With your edit do you still want to change the default?

Reply:
If we do the change, yes, but I think we can handle it separately.